opm/autodiff/StandardWellsDense.hpp

```diff
@@ -69,7 +72,9 @@ enum WellVariablePositions {
         template<typename FluidSystem, typename BlackoilIndices>
         class StandardWellsDense {
         public:
-
+            // --------- Ebos property system stuff -------
+            typedef typename TTAG(EclFlowProblem) TypeTag;
+            typedef typename GET_PROP_TYPE(TypeTag, ElementContext)    ElementContext;
```

Comment:
This is a question for @andlaus: is it possible to make this a little easier to read with the modern `using` syntax, i.e. `using TypeTag = TTAG(EclFlowProblem)`? Also, why not give the TypeTag a more meaningful name in the context it is used, such as `Problem` here for example?